index.html

```diff
@@ -0,0 +1,69 @@
+<html>
+  <head>
+    <link rel="stylesheet" type="text/css" href="index.css">
```

Comment:
I don't think it's a valid HTML document without a `<title>`. Also, `type="text/css"` isn't required for HTML5.